lib/pbench/test/unit/server/query_apis/test_controllers_list.py

```diff
@@ -20,32 +20,9 @@ def _setup(self, client):
             pbench_endpoint="/controllers/list",
             elastic_endpoint="/_search?ignore_unavailable=true",
             payload={"user": "drb", "start": "2020-08", "end": "2020-10"},
-            empty_es_response_payload=Commons.empty_es_response_payload,
+            empty_es_response_payload=Commons.EMPTY_ES_RESPONSE_PAYLOAD,
```

Comment:
I'd still prefer to make this an omittable kwarg on the subclass "constructor" rather than having the subclasses know the superclass attribute name.

Reply:
I'm ambivalent about the omittable kwarg approach -- it's pretty subtle.

An alternative would be to use `self` here instead of `Commons`:  if the subclass doesn't define `EMPTY_ES_RESPONSE_PAYLOAD` as a class attribute, then it will inherit the superclass definition (without referencing it by name) which is all the right things, but the subclass can still override the definition or provide another value.

Reply:
I no longer remember clearly what I thought the wrinkles were....  However, it seemed like the approach included a superclass attribute for a default and a subclass attribute for a default, and the initialization got one behavior if it omitted the kwarg and a different behavior if it specified is as `None`, while preserving the option of specifying a non-default value either using a predefined value (e.g., one of the attributes) or a literal...and, it struck me at the time as though the obvious/straightforward code did not necessarily produce the naively-expected result.

I think I was thinking that if the kwarg were omitted, then the expected result would be `None`, which led to the idea that the subclass code would have to specify the superclass attribute to get what we were thinking would be "the default" value...and, that's not really different from what we ended up with without using a kwarg.

But, anyway, as you say, let's move on.